cmake/projects/freetype/hunter.cmake

```diff
@@ -7,6 +7,17 @@ include(hunter_cacheable)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
+    PACKAGE_NAME
+    freetype
+    VERSION
+    "2.9.1"
+    URL
+    "http://download.savannah.gnu.org/releases/freetype/freetype-2.9.1.tar.gz"
```

Comment:
This package need to be [hunterized](https://docs.hunter.sh/en/latest/reference/terminology/hunterize.html) first:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake-dependencies.html
* https://docs.hunter.sh/en/latest/creating-new/patch.html

I've create branch `hunter-2.9.1`:
* https://github.com/hunter-packages/freetype/tree/hunter-2.9.1

Please send patches there from previous `hunter-2.4.12`:
* https://github.com/hunter-packages/freetype/commits/hunter-2.4.12